webapp/src/Controller/Jury/ExecutableController.php

```diff
@@ -39,9 +40,29 @@ public function __construct(
         protected readonly EventLogService $eventLogService
     ) {}
 
+    private function checkUnused(Executable $e): bool
+    {
+        foreach(['compare', 'run', 'full_debug'] as $config_script) {
+            if ($e->getExecid() === (string)$this->config->get('default_' . $config_script)) {
```

Comment:
Do we need a null check here? In case someone uses an old database?

Reply:
I think a null check would not catch the exception for that case. So I'll wrap this in a `try..catch`